form/form_collections.rst

```diff
@@ -557,7 +557,21 @@ you will learn about next!).
     The trick is to make sure that the single "Task" is set on each "Tag".
     One easy way to do this is to add some extra logic to ``addTag()``,
     which is called by the form type since ``by_reference`` is set to
-    ``false``::
+    ``false``.
+    
+    For a one-to-many relationship you just need to add one line to ``Task``:
```

Comment:
This is misleading, in fact it's the opposite here (i.e. many-to-one). Maybe we should just manage this using inline comments like this:

```php
public function addTag(Tag $tag)
{
    // for a many-to-many association
    $tag->addTask($this);

    // for a many-to-one association
    $tag->setTask($this);

    $this->tags->add($tag);
}
```

Reply:
Yeah, good idea, done that - see https://github.com/symfony/symfony-docs/pull/9259/files#diff-3dccfe528cd4fc9a1dd946f388e5956b